ci/apache_84_57/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  mysql:
+    extends:
+      file: ../compose-shared.yml
+      service: mysql
+    image: mysql:5.7
+    command:
+    - mysqld
+    - --character-set-server=utf8mb4
+    healthcheck:
+      disable: true
+  openemr:
+    extends:
+      file: ../compose-shared.yml
+      service: openemr
+    image: openemr/openemr:flex-3.21
+    depends_on:
+    - mysql
```

Comment:
If your editor has support or a plugin for .editorconfig files, then it should automatically clean up things like this missing trailing newline.